server/gokbg3/src/integration-test/resources/journal_prices_test.json

```diff
@@ -0,0 +1,67 @@
+[
+   {
+      "OAStatus": "",
+      "volumeNumber": "null",
+      "historyEvents": [],
+      "imprint": "",
+      "editionStatement": "null",
+      "editStatus": "In Progress",
+      "defaultAccessURL": "",
+      "dateFirstOnline": "",
+      "medium": "Journal",
+      "type": "Serial",
+      "shortcode": "",
+      "issuer": "",
+      "publisher_history": [{
+         "endDate": "",
+         "name": "S. Karger",
+         "startDate": "",
+         "status": "Active"
+      }],
+      "firstEditor": "null",
+      "firstAuthor": "null",
+      "dateFirstInPrint": "",
+      "identifiers": [
+         {
+            "type": "zdb",
+            "value": "2256676-4"
+         },
+         {
+            "type": "issn",
+            "value": "0001-5547"
+         },
+         {
+            "type": "eissn",
+            "value": "1938-2650"
+         },
+         {
+            "type": "ezb",
+            "value": "150346"
+         }
+      ],
+      "monographEdition": "null",
```

Comment:
Monograph fields may be omitted entirely for `type: "Serial"`. Otherwise `null` would probably be the desired value here instead of the String value.